user/enterprise/high-availability.md

```diff
@@ -0,0 +1,53 @@
+
+## Overview of Installation
+
+The platform installation is quite similar to the standard [Enterprise installation](/user/enterprise/installation#Setting-up-the-Travis-CI-Enterprise-Platform), and the [worker installation is identical](#Installing-the-Worker-in-High-Availability-Mode). However, there are some additional [system prerequisites](/user/enterprise/prerequisites/#High-Availability-Mode), which mean that to install in HA mode, you will need the following:
+ * 3+ **16 gigs of RAM, 8 CPUs, 40GB HDD**, i.e. `c4.2xlarge` with a 40GB HDD. - 2+ for the VMs running the Platform, and 1+ for the VMs running the Worker
+ * [Redis](https://redis.io/), [RabbitMQ](https://www.rabbitmq.com/), 
+and [Postgres](https://www.postgresql.org/) instances
+ * [GitHub OAuth app](/user/enterprise/prerequisites#OAuth-App), [trial license](/user/enterprise/prerequisites#License) -- enabled for HA
+ * Internet connection -- note, this installation is _not_ airgapped by default. Let [us](mailto:enterprise@travis-ci.com) if you are interested in one. 
+
+ Once you have all these things, you should be able and ready to get started.
+
+## Installing the Platform in High Availability Mode
+
+HA is configured entirely on the Enterprise platform instance, but installing an HA platform is quite similar to installing a standard platform. The steps for HA are as follows. 
+
+1. Contact [enterprise@travis-ci.com](mailto:enterprise@travis-ci.com?subject=HA%20Installation) to have your Enterprise license configured for HA mode. 
+1. Setup your [platform instance per the standard installation steps](/user/enterprise/installation#Setting-up-the-Travis-CI-Enterprise-Platform)
+1. Sign into your Admin Dashboard (at `your-domain.tld:8800`)
+1. Go to "Settings" and click "Enable HA"
+1. Paste in the urls where you have [Postgres](https://www.postgresql.org/), [Redis](https://redis.io/), and [RabbitMQ](https://www.rabbitmq.com/) hosted. The connection strings should be in the format of:
+```
+postgres://user:password@url:port
+redis://user:password@url:port
+amqps://user:password@url:port
+```
+1. Optional: Upload a RabbitMQ Client Certificate (`.crt`). This allows RabbitMQ to use TLS. 
+1. Scroll down to the bottom of the page, "Save" and restart
+
+Once your first platform instance is fulling configured, you should be able to see the UI and request a build -- your build will only run correctly though if a worker is installed. Try out your new platform, and [please let us know](mailto:enterprise@travis-ci.com?subject=HA%20Troubleshooting) if you have questions. 
+
+### Adding More Platform Installations
+
+We recommend at least two Platform containers for HA mode. Installation for additional enterprise containers work the same as the first [Platform instance is installed](/user/enterprise/installation#Setting-up-the-Travis-CI-Enterprise-Platform). 
```

Comment:
might be more direct to say something like

"install more enterprise containers  the same way as you installed the first" or something like that